plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchColumnHandle.java

```diff
@@ -27,16 +28,19 @@
 {
     private final String name;
     private final Type type;
+    private final IndexMetadata.Type rawType;
     private final boolean supportsPredicates;
 
     @JsonCreator
     public ElasticsearchColumnHandle(
             @JsonProperty("name") String name,
             @JsonProperty("type") Type type,
+            @JsonProperty("rawType") IndexMetadata.Type rawType,
```

Comment:
Since IndexMetadata.Type isn't in our control can we ensure that it round-trips properly through JSON serialisation and deserialisation?